certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -80,15 +81,14 @@ def test_prepare_version(self, mock_exe_exists, _):
         self.assertRaises(
             errors.NotSupportedError, self.config.prepare)
 
-    @mock.patch("certbot_apache.parser.ApacheParser")
     @mock.patch("certbot_apache.configurator.util.exe_exists")
-    def test_prepare_old_aug(self, mock_exe_exists, _):
+    def test_prepare_old_aug(self, mock_exe_exists):
```

Comment:
I think we should move this test to `parser_test.ParserInitTest` and check that the class blows up during `__init__` if `check_aug_version` fails. It'd simplify this test a bit.